src/ngraph_builder.cc

```diff
@@ -1810,6 +1810,18 @@ static Status TranslateDepthwiseConv2dNativeOp(
   return Status::OK();
 }
 
+static Status TranslateBroadcastDistributedOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input));
+
+  auto ng_broadcast_distributed =
+      ConstructNgNode<ng::op::BroadcastDistributed>(op->name(), ng_input);
+  SaveNgOp(ng_op_map, op->name(), ng_broadcast_distributed);
+  return Status::OK();
```

Comment:
since this looks like a direct one-to-one map, please consider using `TranslateUnary`. Please see "Abs" translation for an example. https://github.com/tensorflow/ngraph-bridge/blob/master/src/ngraph_builder.cc#L4794

Reply:
great idea, used